pandas/core/frame.py

```diff
@@ -3456,7 +3456,32 @@ def _iter_column_arrays(self) -> Iterator[ArrayLike]:
         for i in range(len(self.columns)):
             yield self._get_column_array(i)
 
+    def _check_deprecated_indexers(self, key):
```

Comment:
can this be a function in `indexing.py`  or similar?

Reply:
Yep done, hadn't had much time on friday and wanted to check at least whether I got every occurrence in the test suite